usmqe/web/tendrl/cluster_work.py

```diff
@@ -0,0 +1,230 @@
+"""
+Description: Simple cluster auxiliary methods and classes
```

Comment:
There are several reasons:

- `clusters` is kind of extension of already known classes, it's a place where I put what those classes have similar. `cluster_work` are methods for working with clusters, later there will be `create_cluster` method etc.
- they cannot be together because of circular relation. If that will be merged then  'A imports B' and 'B imports A'. I admit that's the main reason, why they are separated.